### PR TITLE
Task fixing broken templates

### PR DESCRIPTION
**Pull request by paganotoni**

### CODE REVIEW

File: internal/v1/genny/resource/templates/use_model/actions/resource-name.go.tmpl [hunk 1]
```diff
@@ -132,9 +132,9 @@ func (v {{.opts.Name.Resource}}Resource) Create(c buffalo.Context) error {
 
       return c.Render(http.StatusUnprocessableEntity, r.HTML("/{{.opts.Name.File.Pluralize}}/new.plush.html"))
     }).Wants("json", func (c buffalo.Context) error {
-      return c.Error(http.StatusUnprocessableEntity, r.JSON({{.opts.Model.VarCaseSingle}}))
+      return c.Render(http.StatusUnprocessableEntity, r.JSON({{.opts.Model.VarCaseSingle}}))
     }).Wants("xml", func (c buffalo.Context) error {
-      return c.Error(http.StatusUnprocessableEntity, r.XML({{.opts.Model.VarCaseSingle}}))
+      return c.Render(http.StatusUnprocessableEntity, r.XML({{.opts.Model.VarCaseSingle}}))
```
Comment (markbates) [inline internal/v1/genny/resource/templates/use_model/actions/resource-name.go.tmpl]: Shouldn't these be rendering the errors, not the widget?

**Review by markbates — CHANGES_REQUESTED** [review]

File: internal/v1/genny/resource/templates/use_model/actions/resource-name.go.tmpl (hunk 1 above)
Comment (markbates) [reply internal/v1/genny/resource/templates/use_model/actions/resource-name.go.tmpl]:
```
return c.Render(http.StatusUnprocessableEntity, r.JSON(verrs))
```

Comment (paganotoni) [reply internal/v1/genny/resource/templates/use_model/actions/resource-name.go.tmpl]: My only idea there was to provide a response in the same language of the request that's being done to the endpoint. IDK if by setting the error Buffalo would automatically handle JSON/XML to render the error.

Comment (markbates) [reply internal/v1/genny/resource/templates/use_model/actions/resource-name.go.tmpl]: that was correct with `Auto`, and true of `HTML`. while you're returning an error status code, you're never doing anything with the validation errors. the user just sees their submitted data back again.

Comment (paganotoni) [reply internal/v1/genny/resource/templates/use_model/actions/resource-name.go.tmpl]: Oh! I just saw that comment with the example. I will take that path:
```
return c.Render(http.StatusUnprocessableEntity, r.JSON(verrs))
```

**Comment by paganotoni** [comment]
@markbates moved to render `verrs` instead of the widget. Thanks!

**Review by markbates — APPROVED** [review]

**Pull request merged**